src/app/organization/single-certificate/single-certificate.component.html

```diff
@@ -16,10 +24,17 @@ <h1 class="text-center title"> <i class="fa fa-pencil"></i> Issue Certificate </
              <label for="email" class="input-label">Template: </label>
              <select class="form-control" formControlName="template">
                <option selected> Basic </option>
+               <option> Certificate </option>
+               <option> Certificate Vittal </option>
+               <option> Certificate Vittal 2 </option>
+               <option> Certificate Ant </option>
+               <option> Certificate Geek </option>
+               <option> Certificate Template </option>
+               <option> Certsid </option>
              </select>
            </div>
           <div class="form-group">
-            <label for="email" class="input-label">Students name: </label>
+            <label for="email" class="input-label">Name: </label>
```

Comment:
I'm not sure, since when it is there but should we have `for="email"` there?
Correct me, if I'm missing something.

Reply:
As I said before, this generation form isn't my work and I've only made some improvements to it but this `for=" email "` does look a bit weird and as there's no object with `id=" email "` so it's unnecessary to put in the file. therefore I'm removing it.

Reply:
Yeah seems legit